private/sass/addons/_newsblog.scss

```diff
@@ -336,9 +336,6 @@
     .text-muted {
         color: $gray;
     }
-    a {
```

Comment:
it means it wasn't added there 😠 
@genti94 I thought you added this everywhere :/